kolibri/core/content/test/test_zipcontent.py

```diff
@@ -99,9 +99,16 @@ def _get_file(self, file_name, base_url=None, **kwargs):
         return generate_zip_content_response(self.environ)
 
     def test_zip_file_url_reversal(self):
+        from kolibri.utils.conf import OPTIONS
+
+        path_prefix = OPTIONS["Deployment"]["ZIP_CONTENT_URL_PATH_PREFIX"]
```

Comment:
Since this option isn't being overridden using [the decorator](https://github.com/learningequality/kolibri/pull/7924/files#diff-38ed1052b92c27b26b69d8a9d87f366c31634293db3f9b0452293f6748a07d8dR36), this will be directly pulling from the local options.ini file (or envvars), which makes the test less predictable. I might be missing something, though.

Reply:
Tests run in an encapsulated `KOLIBRI_HOME` so this should be safe, it won't be pulling from any previously defined local state.